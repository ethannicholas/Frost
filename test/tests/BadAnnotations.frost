@foo
@class
@private
@protected
@package
@class
@override
@implicit
@weak
@external
@pre(true)
@preOr(true)
@post(true)
@postAnd(true)
class Test {
}
