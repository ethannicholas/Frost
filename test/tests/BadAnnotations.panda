@foo
@class
@private
@protected
@class
@override
class Test {
}
