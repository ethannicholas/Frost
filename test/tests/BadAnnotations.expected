BadAnnotations.frost:1:1: error: unrecognized annotation '@foo'
BadAnnotations.frost:6:1: error: duplicate annotation
BadAnnotations.frost:15:1: error: classes may not be annotated with '@protected'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@class'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@override'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@external'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@implicit'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@weak'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@pre'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@preOr'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@post'
BadAnnotations.frost:15:1: error: classes may not be annotated with '@postAnd'
BadAnnotations.frost:15:1: error: conflicting access modifier annotations
13 errors
